Remove fixed height so QR code can grow based on username length

iandunn opened the pull request:
Fixes #163 

Otherwise a longer username would overflow the container and overlap surrounding elements.

See https://github.com/WordPress/two-factor/blob/654ebb75024c21c116b4155e7254e143c9f64413/providers/class-two-factor-totp.php#L234

outdoor2kode reviewed (approved):
Thanks for the fix. It works for me. 👍

iandunn committed:
Remove fixed height so QR code can grow based on username length

Otherwise a longer username would overflow the container and overlap surrounding elements.

See https://github.com/WordPress/two-factor/blob/654ebb75024c21c116b4155e7254e143c9f64413/providers/class-two-factor-totp.php#L234

Co-Authored-By: Steven Dufresne <steve.dufresne@automattic.com>
